views/index.erb

```diff
@@ -0,0 +1,23 @@
+<%if !@name%>
```

Comment:
Interesting use of `if-else` block in ERB to keep the name entering and multiplayer option screens in one file.